### DISCUSSION TITLE
wHeN wiLL  yOu AdD OsPrEy AnD aPaChE????????

### DISCUSSION THREAD

**Discussion by Dragoteryx** [VJ-HLR-Developers/Half-Life-Resurgence#32] [Q&A] [answered]
when

**Comment by DrVrej** [comment, marked as answer]
When Osprey finds his way home, he wandered off earlier today. We have called the officials to start a search for him but no luck so far, it might takes years to find him in that forest.... Join us every Sunday as we pray for his safe return... 🙏
